tests/integration/test_data/gomod_multiple_modules_missing_checksums/fetch_deps_sha256sums.json

```diff
@@ -0,0 +1,37 @@
+  "gomod/pkg/mod/cache/download/golang.org/x/text/@v/v0.0.0-20170915032832-14c0d48ead0c.ziphash": "sha256:cdc3d43d1d782705a91e8153f593bffeac4453d00e288e06c3c47d2c143a0d2a",
+  "gomod/pkg/mod/cache/download/rsc.io/quote/@v/list": "sha256:59ca590701f9791fd6b84edf03d661d00324590e1887e9f324618cb5e1658ea5",
+  "gomod/pkg/mod/cache/download/rsc.io/quote/@v/v1.5.1.info": "sha256:55b41bf69bd5b874eb0c7a453fdd1ecffbe79ce818b77476ba7a73eeec961f53",
+  "gomod/pkg/mod/cache/download/rsc.io/quote/@v/v1.5.1.lock": "sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+  "gomod/pkg/mod/cache/download/rsc.io/quote/@v/v1.5.1.mod": "sha256:b167e79090d79e4b5738995c62a2186548fc33c919d6b4da1d36cc8c7c1e40c1",
+  "gomod/pkg/mod/cache/download/rsc.io/quote/@v/v1.5.1.zip": "sha256:4ecd78a6d9f571e84ed2baac1688fd150400db2c5b017b496c971af30aaece02",
+  "gomod/pkg/mod/cache/download/rsc.io/quote/@v/v1.5.1.ziphash": "sha256:78da26907e018785c4c96470505d415935147c91180d2627fd634b3f6169e83e",
+  "gomod/pkg/mod/cache/download/rsc.io/quote/@v/v1.5.2.info": "sha256:ca2797bd6559ec66fa2ab170f1c23fc9b1bbbc806890c2e2127f6c89c33e3ab6",
+  "gomod/pkg/mod/cache/download/rsc.io/quote/@v/v1.5.2.lock": "sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+  "gomod/pkg/mod/cache/download/rsc.io/quote/@v/v1.5.2.mod": "sha256:b167e79090d79e4b5738995c62a2186548fc33c919d6b4da1d36cc8c7c1e40c1",
+  "gomod/pkg/mod/cache/download/rsc.io/quote/@v/v1.5.2.zip": "sha256:643fcf8ef4e4cbb8f910622c42df3f9a81f3efe8b158a05825a81622c121ca0a",
+  "gomod/pkg/mod/cache/download/rsc.io/quote/@v/v1.5.2.ziphash": "sha256:6d8d84aa276b56bc81c636ee19ebdc0ebc23af7ed5af354050912be1dc1d0f0a",
+  "gomod/pkg/mod/cache/download/rsc.io/sampler/@v/list": "sha256:1c6e945d74b4dc17923055e616b0f50f46d05ba0c4d9d080a481370905d3e614",
+  "gomod/pkg/mod/cache/download/rsc.io/sampler/@v/v1.3.0.info": "sha256:7421e51c069e136de8ede93e929238027fd9256f3aa4614f79609835585f9d8d",
+  "gomod/pkg/mod/cache/download/rsc.io/sampler/@v/v1.3.0.lock": "sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+  "gomod/pkg/mod/cache/download/rsc.io/sampler/@v/v1.3.0.mod": "sha256:a2ae6e8540a40b85e57703cc618b99c5b1075386d958e8999a0e2d4c02f3a705",
+  "gomod/pkg/mod/cache/download/rsc.io/sampler/@v/v1.3.0.zip": "sha256:da202b0da803ab2661ab98a680bba4f64123a326e540c25582b6cdbb9dc114aa",
+  "gomod/pkg/mod/cache/download/rsc.io/sampler/@v/v1.3.0.ziphash": "sha256:ee5ed9d268a6fb84db6e6f237c214c42f2d9ef7a721f4c28083883234ddadb23",
+  "gomod/pkg/mod/cache/download/sumdb/sum.golang.org/lookup/golang.org/x/text@v0.0.0-20170915032832-14c0d48ead0c": "sha256:c73b55d8ff7744fb5468acc1042bd8e135d46810dd2f4349717e8aa8717cc195",
+  "gomod/pkg/mod/cache/download/sumdb/sum.golang.org/lookup/rsc.io/sampler@v1.3.0": "sha256:aea7de67313bc7e4a7bd7b33ee7c0ecfb8ac2c97fa3995849892b128b8b7f690",
```

Comment:
Curious. We end up downloading some stuff from the GOSUMDB when the checksums are missing in go.sum

Reply:
...and the checksums are not stable? Or different in CI compared to my machine?

```
  E             -  'gomod/pkg/mod/cache/download/sumdb/sum.golang.org/lookup/golang.org/x/text@v0.0.0-20170915032832-14c0d48ead0c': 'sha256:c73b55d8ff7744fb5468acc1042bd8e135d46810dd2f4349717e8aa8717cc195',
  E             ?                                                                                                                           ^^^ ^^^^    ^^^  ^    ^^ ^ ---------   ^^^^ --------------------
  E             +  'gomod/pkg/mod/cache/download/sumdb/sum.golang.org/lookup/golang.org/x/text@v0.0.0-20170915032832-14c0d48ead0c': 'sha256:2eb921fff58e7850038432876b8230eab90513672d588eac2976e0b2681489af',
```

Reply:
Made the check less strict. LMK what you think